### PR TITLE
[API] trivial: move get_resource / get_module handlers to state.rs

### PR DESCRIPTION
**Pull request by msmouse**
## Motivation

Just realized that's how it should be organized. (See events.rs)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?
Y
## Test Plan
existing coverage

### CODE REVIEW

File: api/src/accounts.rs [hunk 1]
```diff
@@ -61,21 +60,6 @@ pub fn get_account_resources(context: Context) -> BoxedFilter<(impl Reply,)> {
         .boxed()
 }
 
-// GET /accounts/<address>/resource/<resource_type>
-pub fn get_account_resource(context: Context) -> BoxedFilter<(impl Reply,)> {
```
Comment (sitalkedia) [inline api/src/accounts.rs:65]: @msmouse - Maybe I am missing something but this API path is still under /accounts - so it makes sense to keep this under accounts.rs?

Comment (msmouse) [reply api/src/accounts.rs:65]: See events.rs -- they are group by functionality or the class powering the endpoints (`struct Events` or `struct State` in this case), not by the path.

**Review by sitalkedia — APPROVED** [review]

**Pull request merged**